src/utils/perfherder/chartJs/perfherderFormatter.js

```diff
@@ -12,8 +12,23 @@ const generateInitialOptions = (meta) => {
   return {
     reverse: higherIsBetter,
     scaleLabel: higherIsBetter ? 'Score' : 'Load time',
+    tooltips: {
+      callbacks: {
+        footer: (tooltipItems, data) => {
+          let delta = 0;
+          if (tooltipItems[0].index > 0) {
+            delta = (data.datasets[tooltipItems[0].datasetIndex].data[tooltipItems[0].index].y
+              - data.datasets[tooltipItems[0].datasetIndex].data[tooltipItems[0].index - 1].y)
+              .toFixed(2);
+            } else {
+              delta = 'n/a';
```

Comment:
Something to think about for future code, is whether you actually need an `else statement`. If you wrote`let delta = 'n/a'` on line 18 instead and then only reassigned it with the `if` condition then the else isn't necessary. This is a way to write conditions in a more concise way but it doesn't apply to every situation.